src/components/Text/text/text.scss

```diff
@@ -68,8 +70,28 @@ $block: '.#{variables.$ns}text';
         @include mixins.text-subheader-3();
     }
     &_ellipsis {
+        display: inline-block;
         @include mixins.overflow-ellipsis();
     }
+
+    // ellipsis lines
+    &_el {
+        display: -webkit-box; // stylelint-disable-line value-no-vendor-prefix
+        -webkit-box-orient: vertical;
+        overflow: hidden;
+        align-self: center;
+        white-space: normal;
+        // by default clamp 2 lines
+        -webkit-line-clamp: 2;
+    }
+
+    @each $size in $supportedLineCountSizes {
```

Comment:
We generate ton of CSS for no reason. Why don't we just let `ellipsisLines` to be a `number` type instead? There is no need in second prop `ellipsisLinesCount`.

Reply:
it's only for inline text util usage, a remove this